CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+## 0.4.0 (Current)
```

Comment:
I suggest to give explicit description on what `Current` means. Is that "currently is under development" or "latest stable release"?

Reply:
Agree to change to "(In Development)"